### PR TITLE
Active records basics

### PR DESCRIPTION
**Pull request by keyur-patel-21**

### CODE REVIEW

File: app/models/product.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Product < ApplicationRecord
+
+    validates :productname, :productprice, :discount, presence: true
+end
```
Comment (DhrutiPatelG) [inline app/models/product.rb:3]: keep spaces:2 and remove extra line

Comment (DhrutiPatelG) [inline app/models/product.rb:3]: keep spaces:2 and remove extra line

File: app/views/home/_header.html.erb [hunk 2]
```diff
@@ -1,30 +1,31 @@
...
         
-      </ul>
-      <form class="d-flex">
-        <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
-        <button class="btn btn-outline-success" type="submit">Search</button>
-      </form>
+      </div>
     </div>
   </div>
 </nav>
```
Comment (DhrutiPatelG) [inline app/views/home/_header.html.erb]: add proper indentation and leave line at end of file

Comment (DhrutiPatelG) [inline app/views/home/_header.html.erb]: add proper indentation and leave line at end of file

File: app/views/products/_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+    <br>
+
+    <div class="actions">
+      <%= form.submit %>
+    </div>
+  </div>
+
+  
+<% end %>
```
Comment (DhrutiPatelG) [inline app/views/products/_form.html.erb:34]: add proper indentation

Comment (DhrutiPatelG) [inline app/views/products/_form.html.erb:34]: add proper indentation

File: app/views/products/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        <td><%= link_to 'Edit', edit_product_path(product),class:"btn btn-primary" %></td>
+        <td><%= link_to 'Destroy', product, method: :delete, data: { confirm: 'Are you sure?' }, class:"btn btn-primary" %></td>
+      </tr>
+    <% end %>
+  </tbody>
+</table>
+
+<br>
+
+
```
Comment (DhrutiPatelG) [inline app/views/products/index.html.erb]: remove extra lines

Comment (DhrutiPatelG) [inline app/views/products/index.html.erb]: remove extra lines

File: test/models/product_test.rb [hunk 5]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class ProductTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end
```
Comment (DhrutiPatelG) [inline test/models/product_test.rb]: do not include test file if not in use

Comment (DhrutiPatelG) [inline test/models/product_test.rb]: do not include test file if not in use

File: app/views/home/_header.html.erb (hunk 2 above)
Comment (keyur-patel-21) [reply app/views/home/_header.html.erb]: done

**Pull request merged**